src/main/java/twilightforest/item/TFRecipes.java

```diff
@@ -316,6 +344,30 @@ public static void registerRecipes() {
                 new Object[] { "###", "# #", "# #", '#', TFItems.fieryIngot });
         GameRegistry
                 .addRecipe(new ItemStack(TFItems.fieryBoots), new Object[] { "# #", "# #", '#', TFItems.fieryIngot });
+
+        // Direct Fiery armor crafting
+        GameRegistry.addRecipe(
+                new ShapelessOreRecipe(
+                        new ItemStack(TFItems.fieryHelm),
+                        new Object[] { new ItemStack(Items.iron_helmet, 1, 0), "fieryEssence", "fieryEssence",
```

Comment:
that also looks pretty vanilla. less clear but probably needs a gregified recipe

Reply:
yea we need to have some gtnh recipes like for the trap doors and the fences before